src/Config/ObjectModel/RuntimeConfig.cs

```diff
@@ -154,6 +156,16 @@ public IEnumerable<DataSource> ListAllDataSources()
         return _dataSourceNameToDataSource.AsEnumerable();
     }
 
+    public bool TryAddEntityPathNameToEntityName(string entityPathName, string entityName)
+    {
+        return _entityPathNameToEntityName.TryAdd(entityPathName, entityName);
+    }
+
+    public bool TryGetEntityNameFromPath(string entityPathName, out string? entityName)
```

Comment:
we now have two methods of the same name, one in runtimeconfig and one in sqlmetdataprovider. Should we really maintain two methods with the same name in different classes that seem to have the same responsibility? Can we refactor this to just be the responsibility of a single method in a single class?

Reply:
fixed this!